src/core/00_github_runner.tf

```
@@ -30,3 +30,96 @@ module "github_runner" {
 
   tags = var.tags
 }
+
+locals {
+  repo_owner = "PagoPA"
+  repo_name  = "io-infra"
+  image_name = "ghcr.io/pagopa/github-self-hosted-runner-azure:beta-runner-v2"
+}
+
+data "azurerm_key_vault" "keyvault" {
+  name                = "io-p-kv-common"
+  resource_group_name = "io-p-rg-common"
+}
+
+data "azurerm_key_vault_secret" "github_pat" {
```

```suggestion
data "azurerm_key_vault_secret" "github_pat_io_infra" {
```
in the future we can have other github_pat for different usage